Models/OrderPart.cs

```diff
@@ -405,7 +405,7 @@ public string CurrencyCode
             get { return Id + " - " +
                 Status + " - " +
                 BillingAddress.Honorific + " " + BillingAddress.FirstName + " " + BillingAddress.LastName + " - " +
-                Currency.GetPriceAsString(Total, CurrencyCode) +
+                CurrencyCode + " " + Total.ToString() +

```

Comment:
I am trying to find a way to revert to using a `GetPriceAsString`, however proper formatting of the currency requires specifying the culture. Here, and in general in static methods like `GetPriceAsString used to be, I don't know the site's culture. I could use the InvariantCulture as default, but that may look odd, as we would end up with two potentially different formatting standards around the store.